src/applications/ivc-champva/10-7959a/config/submitTransformer.js

```diff
@@ -29,15 +29,15 @@ export default function transformForSubmit(formConfig, form) {
   });
   copyOfData.medicalUpload = medicalUpload;
 
-  const primaryEOB = copyOfData?.primaryEOB?.map(el => {
-    return { ...el, documentType: 'EOB' };
+  const primaryEob = copyOfData?.primaryEob?.map(el => {
+    return { ...el, documentType: 'Eob' };
```

Comment:
All keys that use `EOB` needed to be updated to use `Eob` so that they are predictable when converted to/from snake_case on the backend.